### ISSUE TITLE
Unable to transfer JSON between ComfyBox & ComfyUI

### ISSUE THREAD

**Issue by WinstonWoof** [space-nuko/ComfyBox#129]
![image](https://github.com/space-nuko/ComfyBox/assets/128506111/91022ffa-32de-48a0-9254-d94c46ffdb57)

It loads & runs its own default plus https://civitai.com/models/111463?modelVersionId=124584 ) but it doesnt load up any existing JSON or PNG files from my local PC either via the Load Button or by dragndrop

Conversely I cant load any ComfyBox Json (or generated images) in ComfyUI (### ComfyUI Revision: 1197 [67be7eb8])

**Comment by Katmoget** [comment]
similar issue, I am unable to load any ComfyBox files in ComfyUI

**Comment by DarkAlchy** [comment]
I am beginning to think this is a dead project and really only made for Linux to begin with.

**Comment by Katmoget** [comment]
yes, it certainly looks like it at the moment, it's a shame it's pretty good otherwise.

**Comment by hobojoker** [comment]
> Conversely I cant load any ComfyBox Json (or generated images) in ComfyUI (### ComfyUI Revision: 1197 [67be7eb8])


Was this ever intended to be a feature? ComfyUI .json will transfer and be used in ComfyBox because comfybox uses ComfyUI as a backend, but Comfybox dramatically changes the .json so I wouldn't expect ComfyBox .json to work in ComfyUi.

**Comment by Katmoget** [comment]
I guess you're right, working as designed. it would be handy as a feature though.

**Comment by Lalimec** [comment]
did you guys tried comy paste, the issue i have is all the input fields in the pasted nodes being turned into a ui element. i want some values to be predetermined but wasnt able to do it automatically.

**Comment by space-nuko** [comment]
> I am beginning to think this is a dead project and really only made for Linux to begin with.

Sorry about the lack of communication, for reasons outside of my control I've recently become absurdly busy with real life and haven't had much time to work on personal projects as much. But if I get some time in the future I intend to fix some of the open issues

The workflow importing feature is kind of WIP because ComfyBox has a different way of saving node data, I did a bit of work to try to improve the feature parity between ComfyBox and ComfyUI recently: https://github.com/comfyanonymous/ComfyUI/pull/1310

Also there isn't yet a way to convert a ComfyBox workflow back into ComfyUI since that would be more involved (how to deal with nodes in ComfyBox but missing in ComfyUI for example), I don't think such a feature would be impossible but there would have to be some compromises made, like you'd have to rebuild the missing parts of the workflow yourself in a way that makes better sense for ComfyUI. And unless ComfyUI supports node events the frontend logic feature of ComfyBox cannot be ported yet

I actually develop and run this project on Windows with `pnpm`, I'll have to clarify the installation instructions since a bunch of people seem confused on how to install. The part that requires the command line isn't actually necessary if all you want to do is run the program since there is a precompiled version in the README (just unzip and run)
